paper/main.tex

```diff
@@ -86,11 +86,11 @@ \section*{Results}
 \begin{figure}[tp]
   \centering
   \includegraphics[width=1\textwidth]{figs/list_learning}
-  \caption{\small \textbf{Naturalistic extensions of classic list-learning memory analyses.} \textbf{A.} The probability of first recall as a function of the serial position of the event in the episode. \textbf{B}.  The probability of recalling each event, conditioned on having most recently recalled the event \textit{lag} events away in the episode.  \textbf{C.} The proportion of participants who recalled each event, as a function of the serial position of the events in the episode.  All panels: error ribbons denote bootstrap-estimated standard error of the mean.}
+  \caption{\small \textbf{Naturalistic extensions of classic list-learning memory analyses.} \textbf{A.} The probability of first recall as a function of the serial position of the event in the episode. \textbf{B}.  The probability of recalling each event, conditioned on having most recently recalled the event \textit{lag} events away in the episode.  \textbf{C.} The proportion of participants who recalled each event, as a function of the serial position of the events in the episode.  All panels: error ribbons denote the bootstrap-estimated 95\% confidence interval.}
   \label{fig:list-learning}
```

Comment:
did you update the figure with this pull request?  or was the caption incorrect before?

Reply:
The caption was incorrect before; it's correct now